mesonbuild/mtest.py

```diff
@@ -261,6 +262,14 @@ def run_single_test(self, wrap, test):
                 if self.options.verbose:
                     print("%s time out (After %d seconds)" % (test.name, timeout))
                 timed_out = True
+            except KeyboardInterrupt:
+                mlog.warning("CTRL-C detected while running %s" % (test.name))
+                kill_test = True
+            except:
```

Comment:
__[Flake8]__

[E722] do not use bare except'


<!-- Comment by [SideCI](https://sideci.com) -->